clevertap-core/src/main/java/com/clevertap/android/sdk/pushnotification/amp/CTPushAmpWorker.kt

```diff
@@ -0,0 +1,16 @@
+package com.clevertap.android.sdk.pushnotification.amp
+
+import android.content.Context
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import com.clevertap.android.sdk.CleverTapAPI
+import com.clevertap.android.sdk.Logger
+
+class CTPushAmpWorker(context: Context, params: WorkerParameters) : Worker(context, params) {
+
+    override fun doWork(): Result {
+        Logger.v("PushAmpWorker is awake")
+        CleverTapAPI.runJobWork(applicationContext)
```

Comment:
do we need asynctask to start in this?

Reply:
This is not required as the worker is already running on a different thread. The db read is synchronised and the network requests are by default made on postAsyncSafely
Changing the same

Reply:
@piyush-kukadiya Can you please check this?
The Worker itself runs on a separate thread and hence postAsync is not required. 
The db read in runJobWork() is synchronised and the network requests are later by default made on postAsyncSafely